### ISSUE TITLE
[solution needed] training with provided maskrcnn config on custom datasets: hmean remains zero

### ISSUE THREAD

**Issue by yue-cheng-wind** [open-mmlab/mmocr#943]
Hi:
I have a text detection datasets which includes ~326 images and 80% are for training and 20% for testing. I adapted the example maskrcnn configuration for the training. I used 50 epochs just to check if training is progressing properly. However, what I found out is that the accuracy remain ~98.0, and each time during the evaluation the result remains

{"mode": "val", "epoch": 50, "iter": 66, "lr": 0.01, "0_hmean-iou:recall": 0.0, "0_hmean-iou:precision": 0.0, "0_hmean-iou:hmean": 0.0}
even after 50 epochs.

Below is my log file:
[20220417_061559.log](https://github.com/open-mmlab/mmocr/files/8501888/20220417_061559.log)

FYI below are my annotation files:
[7fd67477-9c1c-4267-b54b-0fc56ee32ad8_train.txt](https://github.com/open-mmlab/mmocr/files/8486609/7fd67477-9c1c-4267-b54b-0fc56ee32ad8_train.txt)
[7fd67477-9c1c-4267-b54b-0fc56ee32ad8_test.txt](https://github.com/open-mmlab/mmocr/files/8486610/7fd67477-9c1c-4267-b54b-0fc56ee32ad8_test.txt)

Could you give me any suggestions to modify the configuration? Please let me know if you need any other information!

Thank you so much for the help!

**Comment by gaotongxiao** [comment]
The learning rate is too small. We used `lr=0.08` on 8 GPUs, and you might try `lr=0.01` on one GPU.

**Comment by yue-cheng-wind** [comment]
Hi @gaotongxiao :
Thank you so much for your reply. I've tried lr = 0.03 and still the hmean remains 0 after 50 epochs; and when I tried lr = 0.06 the loss became extremely large in early epochs. In this scenario, should I stick to some relatively small value as lr = 0.03 and wait for longer epochs?

**Comment by yue-cheng-wind** [comment]
@gaotongxiao Hey Tong, could you provide any further suggestions?

**Comment by gaotongxiao** [comment]
I'm not sure as well. The loss seems reasonably low but the model fails on the validation set. You can try evaluating the latest checkpoint on the training set and see if the model works on the training set.

**Comment by yue-cheng-wind** [comment]
> I'm not sure as well. The loss seems reasonably low but the model fails on the validation set. You can try evaluating the latest checkpoint on the training set and see if the model works on the training set.

It's not. Thank you all the same for the reply.
